recipes-support/sbin-path-helper/sbin-path-helper_0.1.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = " Add all sbin dirs to $PATH for root user"
```

Comment:
Extra espace before 'Add', probably good to also remove $ from PATH here to avoid it being converted during parsing.